Documentation/ApiOverview/PasswordHashing/Index.rst

```diff
@@ -277,7 +277,7 @@ Example implementation for TYPO3 frontend::
    // The stored password hash from database
    $passwordHash = 'YYY';
    $success = GeneralUtility::makeInstance(PasswordHashFactory::class)
-       ->get($saltedPassword)
+       ->get($saltedPassword, $mode)
```

Comment:
This should be a literal `'FE'` or `'BE'`, otherwise it's unclear what the values can be.

Reply:
.

Reply:
Should I replace the `$mode` with a string, or write a comment such as `// $mode can be 'FE' or 'BE'`

Reply:
Yeah, in this case it's probably easier if you add the missing `$mode` assignment above and append such a comment. 👍

Reply:
Done :)